changes.d/5809.fix.d

```diff
@@ -0,0 +1 @@
+The Workflow-state command now handles task status and outputs equally.
```

Comment:
```suggestion
The `cylc workflow-state` command now polls for task status (like it does for outputs).
```

Reply:
It's really the other way around but not quite as simple as that, so I was trying to avoid the specifics.